pages/general-information-and-resources/san-francisco.md

```diff
@@ -405,6 +392,18 @@ the health unit is 415-556-2766.
 
 The Health Unit also includes an all-gender bathroom.
 
+### Bike parking
+
+Bike parking is next to the loading dock, entered from the far corner at
+McAllister and Leavenworth.
+
+To request bike parking access: Fill out
+[this form](https://docs.google.com/document/d/18N-RSYzyQ2s9orhgsmIce2aOSKvayI8f433oMmI1Dw4/)
```

Comment:
personally I like making link text as descriptive as possible, so I'd do something like this and maybe rework/combine with the preceding line 

```suggestion
[the bike access request form](https://docs.google.com/document/d/18N-RSYzyQ2s9orhgsmIce2aOSKvayI8f433oMmI1Dw4/)
```